internal/api/grpc/user/v2/totp.go

```diff
@@ -35,3 +35,11 @@ func (s *Server) VerifyTOTPRegistration(ctx context.Context, req *user.VerifyTOT
 		Details: object.DomainToDetailsPb(objectDetails),
 	}, nil
 }
+
+func (s *Server) RemoveTOTP(ctx context.Context, req *user.RemoveTOTPRequest) (*user.RemoveTOTPResponse, error) {
+	objectDetails, err := s.command.HumanRemoveTOTP(ctx, req.GetUserId(), "")
```

Comment:
the command does not check for permission yet (since it was only called by auth API or pre checked mgmt API users)

Reply:
added the permission check, it's defined as "user.write" permission in the other top functions